keps/sig-node/1287-in-place-update-pod-resources/README.md

```diff
@@ -629,6 +637,203 @@ TODO: Identify more cases
 - No major bugs reported for three months.
 - Pod-scoped resources are handled if that KEP is past alpha
 
+## Production Readiness Review Questionnaire
+
+<!--
+
+Production readiness reviews are intended to ensure that features merging into
+Kubernetes are observable, scalable and supportable; can be safely operated in
+production environments, and can be disabled or rolled back in the event they
+cause increased failures in production. See more in the PRR KEP at
+https://git.k8s.io/enhancements/keps/sig-architecture/20190731-production-readiness-review-process.md.
+
+The production readiness review questionnaire must be completed for features in
+v1.19 or later, but is non-blocking at this time. That is, approval is not
+required in order to be in the release.
+
+In some cases, the questions below should also have answers in `kep.yaml`. This
+is to enable automation to verify the presence of the review, and to reduce review
+burden and latency.
+
+The KEP must have a approver from the
+[`prod-readiness-approvers`](http://git.k8s.io/enhancements/OWNERS_ALIASES)
+team. Please reach out on the
+[#prod-readiness](https://kubernetes.slack.com/archives/CPNHUMN74) channel if
+you need any help or guidance.
+
+-->
+
+### Feature Enablement and Rollback
+
+_This section must be completed when targeting alpha to a release._
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [x] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name: InPlacePodVerticalScaling
+    - Components depending on the feature gate: kubelet
```

Comment:
Since kubelet has a feature flag, this definitely will need the Upgrade/Downgrade Strategy and Version Skew Strategy sections filled out. Can you please add those two required sections? See here in the template: https://github.com/kubernetes/enhancements/tree/master/keps/NNNN-kep-template#upgrade--downgrade-strategy

Additionally, is kubelet the only component with a feature flag? The KEP also mentions the API Server and Scheduler as well as API changes.

Reply:
Fixed. kube-apiserver and kube-scheduler also use feature-gate. (I botched copy-paste-fix job in taking the PRR from related CRI KEP that I filled out in January while Tim & I were still updating this KEP). 

I'll revisit the upgrade/downgrade strategy to ensure it still works with this updated KEP, but I doubt it will need any change from the[ implementation for the previous design](https://github.com/kubernetes/kubernetes/pull/92127) that Jordan Liggitt had reviewed.

Reply:
The PRR section questions for handling version skew does not specifically cover/capture the down-level client support required here. But the KEP does mention how this is handled in the [Risks and Mitigations](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/1287-in-place-update-pod-resources#risks-and-mitigations) section and we have [planned tests](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/1287-in-place-update-pod-resources#backward-compatibility-and-negative-tests) to cover it. 
I have manually tested it in the implementation of the earlier design and we do handle the case.

Please let me know if I missed a scenario.